.ci/generate_bear_requirements.py

```diff
@@ -155,18 +206,32 @@ def get_pip_requirements(requirements):
     pip_reqs, npm_reqs, gem_reqs = (
         get_all_requirements(get_all_bears(bear_dirs)))
 
-    requirements = {}
-    requirements['overrides'] = 'coala-build.yaml'
```

Comment:
why remove this?

If the file is being regenerated from scratch, without any existing file, this needs to be there.

Reply:
Yep, missed this, removed this for testing.